stdlib/universe/mode.go

```diff
@@ -0,0 +1,519 @@
+package universe
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux/values"
```

Comment:
Can you ensure the imports are grouped correctly? Standard library imports are grouped together and all other imports are grouped together.